kickstarts/base.ks.erb

```diff
@@ -1,5 +1,8 @@
 # Kickstart file automatically generated by anaconda.
+
 # Run the text based installer to see dependency info in screenshots
+text
```

Comment:
GUI mode was easier to monitor/view/debug/continuing interaction with installer.  (was living in that mode for a while going from 6.x to 7.1).  So if it's not too expensive image size wise, I'd prefer to keep it.  Thanks.